trainerData.js

```diff
@@ -1,11 +1,11 @@
 module.exports = [
-    { id: 1, name: 'Brock', pokemonIds: [74, 95] },
-    { id: 2, name: 'Sabrina', pokemonIds: [64, 122, 49, 65] },
-    { id: 3, name: 'Koga', pokemonIds: [109, 110, 109, 89] },
-    { id: 4, name: 'Giovanni', pokemonIds: [112, 51, 31, 34, 113] },
-    { id: 5, name: 'Blaine', pokemonIds: [58, 77, 78, 59] },
-    { id: 6, name: 'Lt. Surge', pokemonIds: [100, 25, 26] },
-    { id: 7, name: 'Erika', pokemonIds: [71, 114, 44] },
-    { id: 8, name: 'Misty', pokemonIds: [120, 121] },
-    { id: 9, name: 'Ash', pokemonIds: [25, 1, 7, 6, 17, 12] },
-]
\ No newline at end of file
+  { id: 1, name: "Brock", pokemonIds: [74, 95, 44] },
```

Comment:
Brock does not have a Gloom! ;)

Reply:
Reeeeally? So it is true!! I need to know pokemons to pass this course :P haha.... I needed this in order to get some variety in my tests.. dunno if we can use someone else, or maybe make up a new trainer.